composer.json

```diff
@@ -15,11 +15,11 @@
     "require": {
         "php": ">=5.5",
         "zendframework/zend-escaper": "~2.5",
-        "zendframework/zend-stdlib": "~2.5"
+        "zendframework/zend-stdlib": "dev-develop as 2.8.0"
```

Comment:
I wonder if we should finish up zend-stdlib and tag it before merging this; would make things simpler. :)

Same goes for zend-config, below.